source/docs/articles/sites/apache-solr.md

```diff
@@ -12,101 +11,18 @@ Apache Solr is a system for indexing and searching site content. Pantheon provid
 

```

Comment:
@bmackinney the remaining content here is assumed to be true for both Drupal and WordPress. Can you confirm? Also, I'm not sure we need an agnostic doc to summarize Solr on Pantheon, but defer to you. I think we can probably move the remnants of this page to the Drupal doc and reiterate the WP doc alongside development of the plugin.